source/includes/sharding-warning.rst

```diff
@@ -0,0 +1,11 @@
+
+.. warning::
```

Comment:
I think we want a blank line between the admonition and the content here.

Reply:
Fixed